roles/pulp_workers/defaults/main.yml

```diff
@@ -2,10 +2,7 @@
 pulp_config_dir: "/etc/pulp"
 pulp_settings_file: "{{ pulp_config_dir }}/settings.py"
 pulp_group: pulp
-pulp_workers:
-  1:
-    state: started
-    enabled: true
-  2:
-    state: started
-    enabled: true
+pulp_workers: 2
+default_worker:
```

Comment:
I imagined we would not be a user option. It would be useful if a user wanted to create a bunch of systemd units but then start them another way. To me that is rare enough and would require extra work from the user that I don't think adding is valuable. That's just my take; others could feel differently.